### PR TITLE
Follow feature fixes

### PR DESCRIPTION
**Pull request by mayhem**
# Summary

* This is a…
    * (x) Bug fix
    * (x) Feature addition
    * ( ) Refactoring
    * ( ) Minor / simple change (like a typo)
    * ( ) Other

Add/remove users from rooms correctly. Remove some debug prints. Add a rudimentary test for the follow feature. Fix problem with going to follow page without being logged in.

# Problem

* JIRA ticket (_optional_): 

# Solution

# Action

### CODE REVIEW

File: listenbrainz/webserver/views/test/test_follow.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+class FollowViewsTestCase(ServerTestCase, DatabaseTestCase):
+    def setUp(self):
+        ServerTestCase.setUp(self)
+        DatabaseTestCase.setUp(self)
+
+        self.log = logging.getLogger(__name__)
+
+        user = db_user.get_or_create(1, 'iliekcomputers')
+        self.user = User.from_dbrow(user)
```
Comment (paramsingh) [inline listenbrainz/webserver/views/test/test_follow.py]: this isn't needed either

File: listenbrainz/webserver/views/test/test_follow.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+
+    def tearDown(self):
+        self.logstore = None
```
Comment (paramsingh) [inline listenbrainz/webserver/views/test/test_follow.py]: doesn't seem needed.

File: listenbrainz/webserver/views/test/test_follow.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import listenbrainz.db.stats as db_stats
```
Comment (paramsingh) [inline listenbrainz/webserver/views/test/test_follow.py]: we should probably only use the imports we need.

File: listenbrainz/webserver/views/test/test_follow.py (hunk 1 above)
Comment (mayhem) [reply listenbrainz/webserver/views/test/test_follow.py]: self.user does actually get used below.

Comment (paramsingh) [reply listenbrainz/webserver/views/test/test_follow.py]: oh ok. right. :+1:

**Pull request merged**